edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/SystemInfo.cs

```diff
@@ -6,12 +6,20 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Core
 
     public class SystemInfo
     {
-        [JsonConstructor]
-        public SystemInfo(string operatingSystemType, string architecture, string version)
+        public SystemInfo(string operatingSystemType, string architecture, string version, string serverVersion, string kernelVersion, string operatingSystem, int numCpus)
         {
             this.OperatingSystemType = operatingSystemType;
             this.Architecture = architecture;
             this.Version = version;
+            this.ServerVersion = serverVersion;
+            this.KernelVersion = kernelVersion;
+            this.OperatingSystem = operatingSystem;
+            this.NumCpus = numCpus;
+        }
+
+        public SystemInfo(string operatingSystemType, string architecture, string version)
```

Comment:
Do we need the older version of the constructor (with fewer params)?

Reply:
There's some k8s code that uses it, so I figured the easiest thing would be to leave it, since the new fields are blank for k8s anyway